seed/php-sdk/any-auth/src/User/UserClient.php

```diff
@@ -19,4 +22,25 @@ public function __construct(
     ) {
         $this->client = $client;
     }
+
+    /**
+     * @param ?array{baseUrl?: string} $options
+     * @returns mixed
+     */
+    public function get(?array $options): mixed
```

Comment:
is the parameter optional even without the `= null`?

Reply:
Great catch - I noticed your update in https://github.com/fern-api/fern/pull/4645 actually resolved this as soon as I sync'd 👍